templates/serverpod_templates/modulename_server/lib/src/protocol/module_class.yaml

```diff
@@ -5,5 +5,7 @@
 
 class: ModuleClass
 fields:
-  name: String
```

Comment:
This is a breaking change for all Serverpod users. Also, it adds quite a bit of verbosity to the yaml files. The idea is to keep them as easy and clean as possible to read.

A solution would be to be able to both specify a field like:

```yaml
data: int
```

or

```yaml
data:
  type: int
  default: 32
```

Reply:
both works

Reply:
i checked for its type that it is string or map and do func based on its type.
i belive this format will also help us to extend some column property  that we may define in sql like. primary key, foreignkey